runtime/oti/jvmtiInternal.h

```diff
@@ -61,6 +61,9 @@ typedef enum {
 
 #define J9JVMTI_UDATA_BITS (sizeof(UDATA) * 8)
 
+#define J9JVMTI_GETVMTHREAD_ERROR_ON_NULL_JTHREAD 0x1
```

Comment:
These are bits so these values are incorrect.

Reply:
Yes, the last one should be 0x4.